collective/editmodeswitcher/tests/test_functional.py

```diff
@@ -15,7 +22,7 @@ def test_toggling_edit_mode(self, browser):
         browser.login().visit()
         self.assertTrue(
             self.is_editable(),
-            'No ".documentEditable" found on site root. Markup changed?')
+            'Selector not found on site root. Markup changed?')
```

Comment:
@tinagerber this looks as if it would not work on Plone 5. Did you boot it up and test whether it works?

Reply:
It works, but css-selector isn't there anymore.